libs/npm/core/src/utils/generateQuery.ts

```diff
@@ -1,15 +1,13 @@
-import { IFilterSettingsModel } from 'tno-core';
+import { IFilterSettingsModel } from 'hooks';
```

Comment:
You have to be explicit here because the package doesn't know how to resolve this friendly name.  Change to `../hooks`

Reply:
hmm it seems working fine here, but anyway it should have been updated now as suggested, thanks